Tests/bootstrap.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Bootstrap file for running the tests.
+ *
+ * Load the PHPCS autoloader and the WPCS PHPCS cross-version helpers.
+ *
+ * @package Yoast\YoastCS
+ * @link    https://github.com/Yoast/yoastcs
+ * @license https://opensource.org/licenses/MIT MIT
+ * @since   0.6.0
+ */
+
+if ( ! defined( 'PHP_CODESNIFFER_IN_TESTS' ) ) {
+	define( 'PHP_CODESNIFFER_IN_TESTS', true );
+}
+
+$ds = DIRECTORY_SEPARATOR;
+
+// Get the PHPCS dir from an environment variable.
+$phpcsDir = getenv( 'PHPCS_DIR' );
+$phpcsComposerDir = implode( $ds, array( dirname( __DIR__ ), 'vendor', 'squizlabs', 'php_codesniffer' ) );
```

Comment:
The implode logic feels like something which should be abstracted to a helper function as it's used more than once.

Aside from that, it may be useful to set a `$phpcsComposerBaseDir = dirname( __DIR__ );` above this line.